examples/exchange/derivatives/12_Trades/example.go

```diff
@@ -7,29 +7,28 @@ import (
 
 	"github.com/InjectiveLabs/sdk-go/client/common"
 	exchangeclient "github.com/InjectiveLabs/sdk-go/client/exchange"
-	spotExchangePB "github.com/InjectiveLabs/sdk-go/exchange/spot_exchange_rpc/pb"
+	derivativeExchangePB "github.com/InjectiveLabs/sdk-go/exchange/derivative_exchange_rpc/pb"
 )
 
 func main() {
-	//network := common.LoadNetwork("mainnet", "k8s")
 	network := common.LoadNetwork("testnet", "lb")
 	exchangeClient, err := exchangeclient.NewExchangeClient(network)
 	if err != nil {
-		panic(err)
+		fmt.Println(err)
```

Comment:
it's better to panic here (if connection failed, after 5 lines it would panic due to nil deference)

Reply:
Ok, but this is how the example was already implemented. I only swapped examples from spot folder to derivative, because the were mixed with each other

Reply:
I will apply this change in the `dev` branch